### DISCUSSION TITLE
Minio data migration necessary?

### DISCUSSION THREAD

**Discussion by globalexport** [seaweedfs/seaweedfs#4890] [Q&A]
Is it necessary/recommended to migrate minio buckets to a more suitable structure in seaweed S3? 
If so, is there a migration guide somewhere to follow along?

**Comment by chrislusf** [comment]
what's the use case and what are the requirements?

**Reply by globalexport** [reply]
Wie have an existing minIO setup for user uploads and document generation, and I am trying to find out what needed to be done in case we would move over to seaweed. I did not find documentation on this topic.

**Reply by chrislusf** [reply]
If so, yes.

**Reply by globalexport** [reply]
Yes to which part? :) 
Is there a migration guide?

**Reply by chrislusf** [reply]
"Minio data migration necessary?" yes.

Every use case is different. But mostly just use some tools to copy files over, and setup credentials.
